src/commands/database/migrations/mark-all-applied.mjs

```diff
@@ -0,0 +1,172 @@
+import ora from 'ora'
+import { defineCommand, runCommand } from 'citty'
+import { consola } from 'consola'
+import { colors } from 'consola/utils'
+import { isCancel, confirm } from '@clack/prompts'
+import { fetchUser, fetchProject, projectPath, getProjectEnv, getMigrationFiles, useDatabaseQuery, createMigrationsTable, createMigrationsTableQuery } from '../../../utils/index.mjs'
+import link from '../../link.mjs'
+import login from '../../login.mjs'
+import { $fetch } from 'ofetch'
+
+export default defineCommand({
+  meta: {
+    name: 'mark-all-applied',
+    description: 'Marks all migration files as applied without running them.',
+  },
+  args: {
+    production: {
+      type: 'boolean',
+      description: 'Mark all migrations as applied on the production environment.',
+      default: false
+    },
+    preview: {
+      type: 'boolean',
+      description: 'Mark all migrations as applied on the production environment.',
+      default: false
+    },
+    local: {
+      type: 'boolean',
+      description: 'Mark all migrations as applied on the local development environment.',
+      default: false
+    },
+    nuxtPort: {
+      type: 'number',
+      description: 'The port of Nuxt development server.',
+      default: 3000
+    },
+    nuxtHostname: {
+      type: 'string',
+      description: 'The hostname of Nuxt development server.',
+      default: 'localhost'
+    }
+  },
+  async run({ args }) {
+    const localMigrations = (await getMigrationFiles()).map(fileName => fileName.replace('.sql', ''))
+    const { query, params } = generateQuery(localMigrations)
+    const total = localMigrations.length
+
+    if (total === 0) {
+      consola.info('No migrations found in `./server/database/migrations`, please create one first.')
+      return process.exit(0)
+    }
+
+    // set local as default if no env is provided
+    if (!args.production && !args.preview && !args.local) {
+      args.local = true
+      consola.info('No environment provided, defaulting to `local`.')
+    }
+
+    const shouldApply = await confirm({
+      message: `Do you want to mark ${colors.blue(total)} migration${total > 1 ? 's' : ''} as applied on ${args.local ? colors.blue('local') : args.production ? colors.green('production') : colors.yellow('preview')}?`,
+      initialValue: true
+    })
+    if (!shouldApply || isCancel(shouldApply)) {
+      return
+    }
+
+    // self hosted
+    if (args.local && process.env.NUXT_HUB_PROJECT_SECRET_KEY && process.env.NUXT_HUB_PROJECT_URL) {
+      consola.info(`Using \`${process.env.NUXT_HUB_PROJECT_URL}\` to apply migrations.`)
+      const spinner = ora(`Marking ${colors.blue(total)} migration${total > 1 ? 's' : ''} as applied on ${colors.blue('local')}...`).start()
+
+      try {
+        const options = { url: process.env.NUXT_HUB_PROJECT_URL, token: process.env.NUXT_HUB_PROJECT_SECRET_KEY }
```

Comment:
As it's still for local, I think it should be `nuxtHostname` and `nuxtPort` still. The project secret key is just needed to auth with the proxy routes.